### DISCUSSION TITLE
buttons vs grid with actions

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#152] [Q&A]
Working on a VSD design and thinking about the differences/advantages of using a button vs a cell in a grid with an action. 

Purpose of the button would be to activate a new page. In this design the buttons would be arrow to toggle back and forth between 6 sets of core symbols. 

I have it as a cell in a grid now (red arrows in picture below), but I am having a hard time working through the steps in the layout to make this work. Any ideas? Am I explaining this clearly?

<img width="1150" alt="Screenshot 2023-01-13 at 9 45 55 AM" src="https://user-images.githubusercontent.com/113473933/212348422-e81867b6-4dcd-4f50-b98d-b16a7d809799.png">

**Comment by gbishop** [comment]
You can do it either way.  For a fixed position thing that doesn't change buttons are fine. Grid cells are a little more powerful because you can easily make them change their appearance or label. 

You can even make a 1x1 grid stand in for a button if you need a button that can update. I think if I was doing that layout the red arrows would be buttons.
